src/CodeCoverage/Generators/HtmlGenerator.php

```diff
@@ -110,4 +113,14 @@ private function parse()
 		}
 	}
 
+	private function getNumberOfNonEmptyLines($file)
+	{
+		$lines = 0;
+		foreach(explode("\n", file_get_contents($file)) as $line) {
```

Comment:
This will not work with `\r\n`. What about `count(file($file, FILE_SKIP_EMPTY_LINES))`